L/Libuuid/build_tarballs.jl

```diff
@@ -3,25 +3,26 @@
 using BinaryBuilder
 
 name = "Libuuid"
-version = v"2.36"
+version = v"2.39.3"
 
 # Collection of sources required to build FriBidi
 sources = [
     ArchiveSource("https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v$(version.major).$(version.minor)/util-linux-$(version.major).$(version.minor).tar.xz",
```

Comment:
I think you need something like
```suggestion
version_string = "2.39.3"
version = VersionNumber(version_string)

# Collection of sources required to build FriBidi
sources = [
    ArchiveSource("https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v$(version.major).$(version.minor)/util-linux-$(version).tar.xz",
```
Currently you're building `2.39`, not `2.39.3`.

Reply:
Thanks!